src/contexts/video-context.js

```diff
@@ -0,0 +1,54 @@
+import axios from "axios";
+import { useReducer, useEffect, useContext, createContext } from "react";
+
+const VideoContext = createContext();
+
+const videoReducer = (videoState, { type, payload }) => {

```

Comment:
(not mandatory. just a suggestion)as you already have a reducer folder already created you can move this function out of context as well.

Reply:
I will do that.